ingestion/src/metadata/orm_profiler/metrics/window/median.py

```diff
@@ -56,10 +57,11 @@ def fn(self):
         return None
 
     @_label
-    def dl_fn(self, data_frame=None):
-        if is_quantifiable(self.col.datatype):
-            return data_frame[self.col.name].median().tolist()
+    def df_fn(self, df=None):  # pylint: disable=invalid-name
+        """Dataframe function"""
+        if is_quantifiable(self.col.type):
+            return df[self.col.name].median().tolist()
```

Comment:
Why are we not casting here to pandas data frame?

Reply:
Must have missed that.